controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -812,6 +815,10 @@ public ModelNode getNoTextDescription(boolean forOperation) {
                 result.get(ModelDescriptionConstants.REQUIRES).add(required);
             }
         }
+        if (referenceRecorder != null) {
+            result.get(ModelDescriptionConstants.CAPABILITY_REFERENCE).set(referenceRecorder.getBaseRequirementName());
```

Comment:
I don't think we need anything more here.
On resource that itself exposes capability there should be all that extra info.
In this case, dependent name is name of the attribute on which this information is present.
Info about dynamic capability i am not sure if that helps in any way. 
Given that later on when there will be need for resolving that capability that will be done by capability registry and not directly.